CHANGELOG.rst

```diff
@@ -3,9 +3,11 @@
 ###############
 
 ********************
- 0.6.0 (unreleased)
+ 0.6.0 (29.04.2022)
 ********************
 
+This release includes major changes to the handling and support of units in the API and ASDF schemas. All classes now support and require quantities where appropriate. Pure unitless numbers are no longer supported.
+
```

Comment:
```suggestion

Furthermore, a new class called `GenericSeries` was added. It provides a common interface to describe coordinate-based data arrays either by discrete values or mathematical expressions. A built-in mechanism lets you derive specialized series with specific requirements.  For more information, have a look [at the new tutorial](https://weldx.readthedocs.io/en/v0.6.0/tutorials/generic_series.html)

```

Since somebody already merged the suggestion while I was editing it for additional content 😄

Reply:
safer to edit directly in the file I guess :wink: